code/modules/jobs/job_types/cargo_technician.dm

```diff
@@ -30,6 +28,15 @@
 
 	lightup_areas = list(/area/quartermaster/qm, /area/quartermaster/qm_bedroom)
 
+/datum/job/cargo_technician/get_access()
+	. = ..()
+	if (SSjob.initial_players_to_assign < LOWPOP_JOB_LIMIT)
+		. |= ACCESS_GATEWAY
+	LOWPOP_GRANT_ACCESS(JOB_NAME_QUARTERMASTER, ACCESS_QM)
+	LOWPOP_GRANT_ACCESS(JOB_NAME_QUARTERMASTER, ACCESS_VAULT)
+	LOWPOP_GRANT_ACCESS(JOB_NAME_SHAFTMINER, ACCESS_MINING)
```

Comment:
I think it needs `LOWPOP_GRANT_ACCESS(JOB_NAME_SHAFTMINER, ACCESS_GATEWAY)`

Reply:
No changes made: Gateway is already assigned at low-populations to all crew members so isn't necessary here.

Reply:
Add it on top